src/prod/utils/EthlyApiFactory.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import EthlyApi, {
+  eth,
+} from 'ethly-api'
+
+const {
+  EthereumAddress,
+  HttpEthereumClient,
+} = eth
+
+/**
+ * Gas from console.
+ * @param account address of account to deploy contract to
+ * @param httpAddress address to open connection on
+ * @returns {Promise<EthlyApi>} created api
+ */
+function deployContract(account: EthereumAddress, httpAddress: string): Promise<EthlyApi> {
```

Comment:
Not sure that we need this method, see comment below

Reply:
How we will deploy contract then?

Reply:
Hmm, okay